src/math/SphericalHarmonics3.js

```diff
@@ -67,16 +67,16 @@ Object.assign( SphericalHarmonics3.prototype, {
 		target.copy( coeff[ 0 ] ).multiplyScalar( 0.282095 );
 
 		// band 1
-		target.addScale( coeff[ 1 ], 0.488603 * y );
-		target.addScale( coeff[ 2 ], 0.488603 * z );
-		target.addScale( coeff[ 3 ], 0.488603 * x );
```

Comment:
BTW: Where did `Vector3.addScale()` come from? AFAICS, this method does not exists...

Reply:
A typo, or maybe it changed, I'm not sure.

Reply:
https://github.com/mrdoob/three.js/commit/01f4ec10581cb7f52173e50758a776bbe3c13038 FYI @WestLangley